microprofile/galleon-common/src/main/resources/license/microprofile-feature-pack-licenses.xml

```diff
@@ -144,6 +144,28 @@
         </license>
       </licenses>
     </dependency>
+    <dependency>
+      <groupId>com.squareup.okhttp3</groupId>
+      <artifactId>okhttp</artifactId>
+      <licenses>
+        <license>
+          <name>Apache License 2.0</name>
+          <url>http://www.apache.org/licenses/LICENSE-2.0</url>
+          <distribution>repo</distribution>
+        </license>
+      </licenses>
+    </dependency>
+    <dependency>
+      <groupId>com.squareup.okio</groupId>
+      <artifactId>okio</artifactId>
+      <licenses>
+        <license>
+          <name>Apache License 2.0</name>
+          <url>http://www.apache.org/licenses/LICENSE-2.0</url>
+          <distribution>repo</distribution>
+        </license>
+      </licenses>
+    </dependency>
```

Comment:
Please put these in alphabetical order by maven GA. Otherwise EAP prod will end up having to fix it.

The pom deps the order doesn't matter except that if they are organized in some matter it's good to maintain that.

Reply:
Done.